### PR TITLE
feat: vim.notify in Neorg log

### PR DESCRIPTION
**Pull request by danymat**

### CODE REVIEW

File: lua/neorg/external/log.lua [hunk 1]
```diff
@@ -26,12 +26,12 @@ local default_config = {
...
-        { name = "info", hl = "None" },
-        { name = "warn", hl = "WarningMsg" },
-        { name = "error", hl = "ErrorMsg" },
-        { name = "fatal", hl = "ErrorMsg" },
+        { name = "trace", hl = "Comment", level = vim.log.levels.TRACE },
+        { name = "debug", hl = "Comment", level = vim.log.levels.DEBUG },
+        { name = "info", hl = "None", level = vim.log.levels.INFO },
+        { name = "warn", hl = "WarningMsg", level = vim.log.levels.WARN },
+        { name = "error", hl = "ErrorMsg", level = vim.log.levels.ERROR },
+        { name = "fatal", hl = "ErrorMsg", level = 5 },
```
Comment (mrossinek) [inline lua/neorg/external/log.lua:34]: What's different about this level? I.e. why not `vim.log.levels.FATAL`? Where is `vim.log.levels` defined?

Comment (p00f) [reply lua/neorg/external/log.lua:34]: `:h vim.log.levels`:
```
        vim.log.levels.DEBUG
        vim.log.levels.ERROR
        vim.log.levels.INFO
        vim.log.levels.TRACE
        vim.log.levels.WARN
```

**Comment by danymat** [comment]
Is this ready to merge ?

**Comment by vhyrro** [comment]
Yup, should be good to merge imo!

**Pull request merged**